### DISCUSSION TITLE
How to remove AppPanelProvider after creating it?

### DISCUSSION THREAD

**Discussion by CodeWithPranta** [filamentphp/filament#8695] [General]
I want to rename my AppPanelProvider to TeacherPanelProvider how to remove it, I deleted this file but now getting error. How I remove using terminal?

**Comment by krekas** [comment]
You need to have at least one default panel provider. So if instead of default `admin` you need `teacher` rename it or create a new one and set it default. Only then deleting the default one should work.

**Comment by VincvanAs** [comment]
After deleting the panel, also remove the panel from your config/app.php providers array
